src/content/zh-TW/2020/performance.md

```diff
@@ -0,0 +1,387 @@
+---
+part_number: II
+chapter_number: 9
+title: 效能
+description: 2020 Web Almanac 的效能章節涵蓋了 <span lang="en">Core WebVitals</span>、<span lang="en">Lighthouse</span> <span lang="en">Performance Score</span>、<span lang="en">First Contentful Paint</span> (FCP) 以及 <span lang="en">Time to First Byte</span> (TTFB)。
```

Comment:
The description is plain text not HTML so can we remove spans from here and revert this back? Sorry not obvious what can take HTML and what can't!

```suggestion
description: 2020 Web Almanac 的效能章節涵蓋了 Core WebVitals、Lighthouse Performance Score、First Contentful Paint (FCP) 以及 Time to First Byte (TTFB)。
```

Reply:
Thanks for catching it! I was not sure if they should be wrapped with `lang=en` so just tried this way first. Let me revert them! Thank you!